BrocoliSe/BrocoliSe/View/Scenes/Onboardings/Onboarding03.swift

```diff
@@ -0,0 +1,109 @@
+        title.textColor = .blueDark
+        title.textAlignment = .center
+        title.numberOfLines = 0
+        return title
+    }()
+    
+    let foodSelector = FoodSelectorComponent(frame: UIScreen.main.bounds)
+    
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        self.isUserInteractionEnabled = true 
+        style()
+        layout()
+    }
+    
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    @objc func targetContiue(_ sender: UIButton) {
+        buttonContinueAction()
+    }
+    
+    func style() {
+        imageView.translatesAutoresizingMaskIntoConstraints = false
+        
+        welcomeText.translatesAutoresizingMaskIntoConstraints = false
+        
+        foodSelector.translatesAutoresizingMaskIntoConstraints = false
+        
+        buttonConfirm.translatesAutoresizingMaskIntoConstraints = false
+    }
+    
+    func layout() {
+        addSubview(imageView)
+        addSubview(welcomeText)
+        addSubview(foodSelector)
+        addSubview(buttonConfirm)
+        
+        NSLayoutConstraint.activate([
+            
+            imageView.heightAnchor.constraint(equalTo: self.widthAnchor, multiplier: 0.5),
+            imageView.centerYAnchor.constraint(equalTo: self.centerYAnchor, constant: -20),
+            imageView.centerXAnchor.constraint(equalTo: self.centerXAnchor, constant: 0),
+            
+            welcomeText.leadingAnchor.constraint(equalToSystemSpacingAfter: self.leadingAnchor, multiplier: 4),
+            self.trailingAnchor.constraint(equalToSystemSpacingAfter: welcomeText.trailingAnchor, multiplier: 4),
+            welcomeText.topAnchor.constraint(equalTo: imageView.bottomAnchor, constant: 20),
+            welcomeText.centerXAnchor.constraint(equalTo: self.centerXAnchor, constant: 0),
+            
+            foodSelector.widthAnchor.constraint(equalTo: self.widthAnchor, multiplier: 0.74),
+            foodSelector.heightAnchor.constraint(equalTo: self.heightAnchor, multiplier: 0.098),
+            foodSelector.topAnchor.constraint(equalTo: welcomeText.bottomAnchor, constant: 10),
+            foodSelector.centerXAnchor.constraint(equalTo: self.centerXAnchor),
+            
+            buttonConfirm.widthAnchor.constraint(equalTo: self.widthAnchor, multiplier: 0.38),
+            buttonConfirm.heightAnchor.constraint(equalTo: self.heightAnchor, multiplier: 0.053),
+            buttonConfirm.topAnchor.constraint(equalTo: foodSelector.bottomAnchor, constant: 20),
+            buttonConfirm.centerXAnchor.constraint(equalTo: self.centerXAnchor)
+        ])
+    }
+    
+}
+
+extension Onboarding03: OnboardingViewControllerProtocol {
+    func buttonContinueAction() {
+        guard let onboardingVC = onboardingVC,
+              let didSendContinue = onboardingVC.didSendContinue else {return}
+        didSendContinue()
+    }
+    
+    
+}
+
```

Comment:
BrocoliSe/BrocoliSe/View/Scenes/Onboardings/Onboarding03.swift#L109
Trailing Newline Violation: Files should have a single trailing newline. (trailing_newline)